packages/binding-http/test/http-server-oauth-tests.ts

```diff
@@ -26,13 +26,6 @@ should();
 class OAuthServerTests {
     private server!: HttpServer;
     async before() {
-        // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
@relu91 the code was there to have a clean test log, right. Do we still need/want it?